modules/prometheus-exporter_oracle/conf/18-dbvnet.yaml

```diff
@@ -0,0 +1,18 @@
+module: oracle
```

Comment:
same here for the tip, I don't know what is the job of this process. if this is obvious for someone who know oracle you can ignore

Reply:
same than above.